### ISSUE TITLE
Update docs about translations

### ISSUE THREAD

**Issue by stsewd** [rtfd/readthedocs.org#3777]
The docs here needs to be updated https://docs.readthedocs.io/en/latest/localization.html according the recent changes on translations (only can add translations where the user is owner)

**Comment by stsewd** [comment]
Reading the current docs, seems ok to me, because there is no mention of adding a project from other user, or nested translations allowed, so, I think with the current UI explains the limitations by itself.

**Comment by RichardLitt** [comment]
Might be worth adding notes so that #3348 and #4152 can be resolved more gracefully in the future.

**Comment by agjohnson** [comment]
Agreed. We need much better docs around this all. I'm really looking forward to making translations a priority on RTD. I think we can do much better at offering first class support for translations. They are obviously more of an after thought currently.

First steps here I think are:

* Describe how to manage translations better on RTD projects
* Describe patterns for this (how to support multiple translations in one repo)
* Explain the sphinx steps better

Next steps on our side are things like Transifex integration, Weblate integration, and automating the Sphinx configuration for multiple languages.

**Comment by ericholscher** [comment]
This is 8 years old, I think we can probably close it?

**Comment by RichardLitt** [comment]
I would close as not actionable. The work of updating is never done, this needs to be more specific.
